hi.js

```diff
@@ -564,7 +564,7 @@ windows.controls.premium1 गाड मोड
 windows.controls.premium2 खुद को नष्ट करे
 windows.controls.premium3 स्माइट
 windows.controls.reset स्थान रिसेट करे (रेस & पार्कोर)
-windows.controls.resetLast Reset To Last Checkpoint (Parkour etc)
+windows.controls.resetLast रिसेट टू लास्ट चेकपॉइंट (Parkour etc)
```

Comment:
no need to convert english statement in hindi. It should be complete, easy translation